src/test/java/com/kodilla/patterns2/facade/OrderFacadeAspectTestSuit.java

```diff
@@ -0,0 +1,39 @@
+package com.kodilla.patterns2.facade;
+
+import com.kodilla.patterns2.facade.api.ItemDto;
+import com.kodilla.patterns2.facade.api.OrderDto;
+import com.kodilla.patterns2.facade.api.OrderFacade;
+import com.kodilla.patterns2.facade.api.OrderProcessingException;
+import org.junit.Test;
+
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.boot.test.context.SpringBootTest;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class OrderFacadeAspectTestSuit{
```

Comment:
dodaj więcej przypadków testowych. 
obczaj sobie bibliotekę assertj. ma fajne api do łapania wyjątków w testach. a to też powinieneś przetestować